Core/Core/Api/GraphQL/Client.GraphqlCleintOperations/Client.StreamOperations.cs

```diff
@@ -9,6 +9,48 @@ namespace Speckle.Core.Api;
 
 public partial class Client
 {
+
+  /// <summary>
+  /// Cheks if a stream exists by id.
+  /// </summary>
+  /// <param name="id">Id of the stream to get</param>
+  /// <returns></returns>
+  public Task<bool> StreamExists(string id)
+  {
+    return StreamExists(CancellationToken.None, id);
+  }
+
+  /// <summary>
+  /// Cheks if a stream exists by id.
+  /// </summary>
+  /// <param name="id">Id of the stream to get</param>
+  /// <returns></returns>
+  public async Task<bool> StreamExists(CancellationToken cancellationToken, string id)
```

Comment:
The method signature you have done does match what we use currently.

But I would like us to consider switching to the more C# recommended system of having the cancellation token as
an optional parameter, instead of having a overload function that uses CancellationToken.None

```cs
public async Task<bool> StreamExists(string id, CancellationToken cancellationToken = default)
```

Reply:
I've been waiting for an opportunity to switch our current functions over to use this. so address some compiler warnings that it generates + reduce the boilerplate in the Client class.

Potentially we want to make new API functions comply?
But there is an argument to keep them consistant

Reply:
This is a only a recommendation from .NET, not a strictly necessarily change 
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1068 

So happy to ignore this for now if you'd like